service.watchdog/resources/language/resource.language.en_GB/strings.po

```diff
@@ -0,0 +1,203 @@
+# Kodi Media Center language file
```

Comment:
can you rename en_GB to en_gb just for consistency? It's not common to use uppercase chars in the language layout folder